components/04-molecules/article&blog-post-elements/author-bio/author-bio-large.html

```diff
@@ -0,0 +1,29 @@
+<div class="list-teaser__footer">
```

Comment:
Ist dieses <div> hier wirklich notwendig?

Reply:
das div hilft bei der horizontalen Positionierung und bringt auch die richtige Font color mit.

Reply:
Die Kalsse `.list-teaser__footer` gehört zu einer anderen Komponente, die nichts mit der Komponente `author-bio` zu tun hat. Daher kann sie hier nicht verwendet werden. Schau dir mal `author-bio-long` an, da kannst du sicherlich das richtige Markup ableiten. Als Faustregel gilt: Im komponentenbasierten Design verwenden wir nichts von anderswo wieder, was essentiell für die eigene Komponente ist. Wir wollen jede Komponente so gut es geht unabhängig von andern halten. Wenn in der Komponente `simple-list-teaser` die Klasse `.list-teaser__foter` verändert wird, ist die `author-bio--large` auch betroffen, und das wollen wir nicht.

Reply:
Ah, okay! Notiere ich mir :)